core/src/main/java/com/graphhopper/routing/util/countryrules/AustriaCountryRule.java

```diff
@@ -0,0 +1,77 @@
+/*
+ *  Licensed to GraphHopper GmbH under one or more contributor
+ *  license agreements. See the NOTICE file distributed with this work for
+ *  additional information regarding copyright ownership.
+ *
+ *  GraphHopper GmbH licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except in
+ *  compliance with the License. You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package com.graphhopper.routing.util.countryrules;
+
+import com.graphhopper.routing.ev.RoadAccess;
+import com.graphhopper.routing.ev.RoadClass;
+import com.graphhopper.routing.util.TransportationMode;
+
+public class AustriaCountryRule implements CountryRule {
+    public final static GermanyCountryRule RULE = new GermanyCountryRule();
+
+    @Override
+    public double getMaxSpeed(RoadClass roadClass, TransportationMode transport, double currentMaxSpeed) {
+        if (!Double.isNaN(currentMaxSpeed) || !transport.isMotorVehicle()) {
+            return currentMaxSpeed;
+        }
+
+        switch (roadClass) {
+            case MOTORWAY:
+                return 130;
+            case TRUNK:
+            case PRIMARY:
+            case SECONDARY:
+            case TERTIARY:
+            case UNCLASSIFIED:
+                return 100;
+            case RESIDENTIAL:
+                return 50;
+            case LIVING_STREET:
+                return 20;
+            default:
+                return Double.NaN;
+        }
+    }
+
+    @Override
+    public RoadAccess getAccess(RoadClass roadClass, TransportationMode transport, RoadAccess currentRoadAccess) {
+        if (currentRoadAccess != RoadAccess.YES) {
+            return currentRoadAccess;
+        }
+
+        if (!transport.isMotorVehicle()) {
+            return RoadAccess.YES;
+        }
+
+        switch (roadClass) {
+            case LIVING_STREET:
+                return RoadAccess.DESTINATION;
+            case TRACK:
+                return RoadAccess.FORESTRY;
```

Comment:
Not subject of this PR but we should discuss this. This blocks all tracks in Austria for cars which is a bit too restrictive IMHO.

Reply:
Ok, please open an issue for this as it also applies to current master.

Reply:
Done: https://github.com/graphhopper/graphhopper/issues/2362